src/lib/module-info.ts

```diff
@@ -7,16 +7,15 @@ import { hasWindowsSlashes, joinPaths, normalizeSlashes, sort } from "../util/ut
 
 import { readFileAndThrowOnBOM } from "./definition-parser";
 
-export default async function getModuleInfo(
-    packageName: string, packageDirectory: string, allEntryFilenames: ReadonlyArray<string>,
-    fs: FS): Promise<ModuleInfo> {
-    const all = await allReferencedFiles(allEntryFilenames, fs, packageDirectory);
+export async function getModuleInfo(packageName: string, all: Map<string, ts.SourceFile>): Promise<ModuleInfo> {
 
     const dependencies = new Set<string>();
     const declaredModules: string[] = [];
     const globals = new Set<string>();
 
-    function addDependency(dependency: string): void {
+    function addDependency(ref: string): void {
+        if (ref.startsWith(".")) return;
```

Comment:
this now happens for all calls to addDependency, which fixes a couple of bogus dependencies I saw on DT.